test/inbox.ts

```diff
@@ -2,13 +2,15 @@
 /* tslint:disable: no-unused-expression */
 import * as chai from 'chai';
 import * as ChaiAsPromised from 'chai-as-promised';
+import ChaiString = require('chai-string');
```

Comment:
`chair-string` actually exports by default the instance to use, and not an object with properties, so we need this.